### PR TITLE
Add chart rate limit configuration support

### LINKED ISSUE #571: Update Deployment Logic with Rate Limit Configs
### Problem

Currently the deployment charts logic is missing rate limit configuration logic.
The env values are in the `.env` but not exposed in the Chart

### Solution

Update Deployment Logic with Rate Limit Configs

### Alternatives

_No response_

### PR DESCRIPTION
**Pull request by Nana-EC**
Signed-off-by: Nana Essilfie-Conduah <nana@swirldslabs.com>

**Description**:
Update Deployment Logic with IP & HBAR Rate Limit Configs
- Update chart configmap yaml with env 
- Map config env values to chart deployment yaml
- Update chart values yaml with default

**Related issue(s)**:

Fixes #571 

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: helm-chart/templates/deployment.yaml [hunk 1]
```diff
@@ -89,7 +89,37 @@ spec:
...
                 name: {{ include "json-rpc-relay.fullname" . }}
                 key: OPERATOR_KEY_MAIN
-                optional: false                               
+                optional: false
+          - name: DEFAULT_RATE_LIMIT
+            valueFrom:
+              secretKeyRef:
+                name: {{ include "json-rpc-relay.fullname" . }}
+                key: DEFAULT_RATE_LIMIT
+                optional: false
```
Comment (rustyShacklefurd) [inline helm-chart/templates/deployment.yaml:98]: Should these be `optional: true` ?

File: helm-chart/templates/deployment.yaml [hunk 2]
```diff
@@ -89,7 +89,37 @@ spec:
               secretKeyRef:
                 name: {{ include "json-rpc-relay.fullname" . }}
                 key: OPERATOR_KEY_MAIN
-                optional: false                               
+                optional: false
+          - name: DEFAULT_RATE_LIMIT
+            valueFrom:
+              secretKeyRef:
```
Comment (rustyShacklefurd) [inline helm-chart/templates/deployment.yaml:95]: `secretKeyRef` should be `configMapKeyRef`

File: helm-chart/templates/deployment.yaml (hunk 1 above)
Comment (Nana-EC) [reply helm-chart/templates/deployment.yaml:98]: Yeah, good call

Comment (Nana-EC) [reply helm-chart/templates/deployment.yaml]: Fixed

File: helm-chart/templates/deployment.yaml (hunk 2 above)
Comment (Nana-EC) [reply helm-chart/templates/deployment.yaml]: Fixed

**Review by rustyShacklefurd — APPROVED** [review]
LGTM

**Review by shemnon — DISMISSED** [review]

**Review by georgi-l95 — APPROVED** [review]

**Pull request merged**